var/spack/repos/builtin/packages/actsvg/package.py

```diff
@@ -50,6 +50,7 @@ class Actsvg(CMakePackage):
     depends_on("boost +program_options", when="+examples")
     depends_on("googletest", when="+examples")
     depends_on("python@3.8:", when="+python")
+    depends_on("py-pybind11@2.10", when="+python @0.4.42:")
```

Comment:
Is the 2.10 version pinning a hard requirement that's documented somewhere, or simply the version that's vendored? Pinning may be affecting compatibility in unified environments.

Reply:
Let me try using an open version range and see if it builds. :+1:

Reply:
Seems to work fine; updated to make the version range open.